client/lib/screen-title/index.js

```diff
@@ -19,26 +19,24 @@ const TITLE_UPDATING_ACTIONS = [
 ];
 
 /**
- * Given a Redux store instance, subscribes to the store, updating the page
+ * A middleware that updating the page
  * when the title state changes. Currently, this dispatches through the legacy
  * Flux actions used elsewhere in the codebase.
  *
  * @param {Object} store Redux store instance
+ * @returns {Function} A configured middleware with store
  */
-export function subscribeToStore( store ) {
-	const dispatch = store.dispatch;
-	store.dispatch = function( action ) {
-		dispatch( ...arguments );
+export const screenTitleMiddleware = store => next => action => {
```

Comment:
- We might want to update the README to indicate that this is middleware.
- We might want to move this to `lib/screen-title-middleware` to match the name of the function.